src/webgl2_resources.rs

```diff
@@ -86,6 +86,15 @@ pub struct WebGL2Pipeline {
     pub color_states: Vec<ColorStateDescriptor>,
     pub depth_stencil_state: Option<DepthStencilStateDescriptor>,
     pub rasterization_state: Option<RasterizationStateDescriptor>,
+    pub scissors_state: Option<ScissorsState>,
```

Comment:
I'm still new to both webgl and Vulkan, but if I got it right, Vulkan pipelines have viewport states attached to pipelines. (https://vulkan-tutorial.com/Drawing_a_triangle/Graphics_pipeline_basics/Fixed_functions#page_Viewports-and-scissors) In order to make the behaviour consistent and prevent scissors affecting drawing of other pipelines, we should store scissors state for webgl pipelines as well.